experiments/standalone/Soil/zettl_sv62.jl

```diff
@@ -0,0 +1,230 @@
+using Plots
+import SciMLBase
+import ClimaTimeSteppers as CTS
+using ClimaCore
+import CLIMAParameters as CP
+using ArtifactWrappers
+using DelimitedFiles: readdlm
+
+using ClimaLand
+using ClimaLand.Domains: Column
+using ClimaLand.Soil
+import ClimaLand
+FT = Float64
+# Define simulation times
+t0 = Float64(0)
+tf = Float64(60 * 60)
+dt = Float64(30)
+
+# Domain
+zmax = FT(0)
+zmin = FT(-1.1)
+nelems = 75
+Δ = FT((zmax - zmin) / nelems / 2)
+soil_domain = Column(; zlim = (zmin, zmax), nelements = nelems)
+
+parameter_data = readdlm(
+    "/Users/katherinedeck/Desktop/code/ClimaLand.jl/experiments/Standalone/Soil/sv_62.csv",
```

Comment:
Ill move these to box when we add this to buildkite. itll be part of this PR too